java/org/contikios/cooja/mspmote/MspMoteType.java

```diff
@@ -316,8 +316,8 @@ public boolean setConfigXML(Simulation simulation,
       }
 
       /* Backwards compatibility: Generate expected firmware file name from source */
-      logger.warn("Old simulation config detected: no firmware file specified, generating expected");
       fileFirmware = getExpectedFirmwareFile(fileSource);
+      logger.warn("Old simulation config detected: no firmware file specified, using '" + fileFirmware + "'");
```

Comment:
Nice to include the expected file name in log output but I think variables should be specified as arguments with log4j 2 to avoid unnecessary creation of strings:

`logger.warn("Old simulation config detected: no firmware file specified, using '{}'", fileFirmware);`